specification/sbi_cove.adoc

```diff
@@ -2077,6 +2137,47 @@ register indexes.
 | SBI_ERR_FAILED          | The operation failed for unknown reasons.
 |===
 
+[#sbi_covg_promote_to_tvm]
+=== Function: COVE Guest Promote to TVM (FID #11)
+[source, C]
+-----
+struct sbiret sbi_covg_promote_to_tvm(unsigned long fdt_addr,
+                                      unsigned long tap_addr);
+-----
+This intrisic is used by a VM to be promoted to a TVM. A VM must not operate as a TVM at the time of this 
+call, otherwise this ABI will return error. It is primarily inteded for COVE deployment models that require 
+single-step TVM creation (e.g., see <<dep3>>). Deployment models that offer multi-step TVM creation 
+(e.g., <<dep1>>) may, but are not required to, support this ABI as an additional mechanism for creating TVMs. 
+
+The `fdt_addr` is the 8-bytes aligned guest physical address of the flattened device tree (FDT). `tap_addr` 
+is the 8-bytes aligned guest physical address of the `TVM attestation payload` used for local attestation. 
+VMs that do not require local attestation (only the remote attestation) must write `0` to `tap_addr`.
+
+The VM must make this call early in the boot process so that the contents of the memory are known and the 
+resulting integrity measurement hashes are deterministic. The TSM receives this call and uses the VM page 
```

Comment:
this sentence is odd - the contents of memory are known to whom? suggest re-state as:

"The VM should issue this call as early in the boot process as possible to minimize changes in contents of memory so that the resulting integrity measurement hashes are deterministic."

Reply:
I modified the text according to your suggestion.